.github/workflows/ci.yml

```diff
@@ -429,6 +429,45 @@ jobs:
         run: |
           ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}
 
+  test_amd_w7900:
+    needs: [setup, build_all]
+    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'test_amd_w7900')
```

Comment:
As discussed, start this job on postsubmit only: https://github.com/iree-org/iree/blob/8ae8aafb827419ce261ba43af5d9b1277049e3cb/build_tools/github_actions/configure_ci.py#L123-L134

This runner type is not stable or available enough for blocking presubmit at the moment (the GPU crashes, requiring a reboot, and we only have one machine). We can land this on postsubmit and monitor from there.

Reply:
Thanks, I added that.

Reply:
Added `ci-extra: test_amd_w7900` to the PR description to run the job on this PR. Please push an empty commit to retrigger CI. Need to see a clean run with the Vulkan tests included too before merging.

Reply:
Sure, just pushed one.